src/com/spikes2212/command/genericsubsystem/GenericSubsystem.java

```diff
@@ -29,6 +31,17 @@ public GenericSubsystem() {
 	 * @param maxSpeed the maximum speed
 	 */
 	public GenericSubsystem(double minSpeed, double maxSpeed) {
+		this.maxSpeed = () -> maxSpeed;
+		this.minSpeed = () -> minSpeed;
```

Comment:
Use the base constructor, pass to it the two lambdas that return the values. Just like in most of our commands